### PR TITLE
Verify opae-c properties tests work on hw

### PR DESCRIPTION
**Pull request by jelonanX**
rebase from PR #845

### CODE REVIEW

File: testing/opae-c/test_props_c.cpp [hunk 1]
```diff
@@ -112,7 +106,7 @@ class properties_p1 : public ::testing::TestWithParam<std::string> {
  *          Then the return value is FPGA_OK<br>
  *          And the field in the token object is set to the known value<br>
  * */
-TEST_P(properties_p1, get_parent01) {
+TEST_P(properties_c_p, get_parent01) {
```
Comment (ibarraza) [inline testing/opae-c/test_props_c.cpp]: This test seems to segfault for me.

Comment (tswhison) [reply testing/opae-c/test_props_c.cpp]: Probably a double-free on parent on line 141.  Isn't parent just one of toks[i] as of 129.  I'd say remove 141 and run valgrind memcheck to confirm it's not leaked.

Comment (jelonanX) [reply testing/opae-c/test_props_c.cpp]: I just ran it on 37 and it all passed.

Comment (tswhison) [reply testing/opae-c/test_props_c.cpp]: I've seen them pass before, but fault in valgrind.  It's worth a shot.

**Comment by ibarraza** [comment]
@tswhison Yes. No errors detected or anything else in the tests.

**Pull request merged**